engines/toltecs/metaengine.cpp

```diff
@@ -199,6 +205,59 @@ SaveStateDescriptor ToltecsMetaEngine::querySaveMetaInfos(const char *target, in
 	return SaveStateDescriptor();
 }
 
+Common::KeymapArray ToltecsMetaEngine::initKeymaps(const char *target) const {
+	using namespace Common;
+	using namespace Toltecs;
+
+	Keymap *engineKeyMap = new Keymap(Keymap::kKeymapTypeGame, "toltecs-default", _("Default keymappings"));
+	
+	Action *act;
+
+	act = new Action(kStandardActionLeftClick, _("Move / Select"));
+	act->setLeftClickEvent();
+	act->addDefaultInputMapping("MOUSE_LEFT");
+	engineKeyMap->addAction(act);
```

Comment:
I recommend a few of the defaults for JOY buttons as well from the now overridden MetaEngine::initKeymaps.